workers/unity/Packages/com.improbable.gdk.gameobjectcreation/GameObjectInitializationSystem.cs

```diff
@@ -47,15 +46,24 @@ private struct RemovedEntitiesData
         private readonly IEntityGameObjectCreator gameObjectCreator;
         private EntityGameObjectLinker linker;
 
-        public GameObjectInitializationSystem(IEntityGameObjectCreator gameObjectCreator)
+        private readonly GameObject workerGameObject;
+
+        public GameObjectInitializationSystem(IEntityGameObjectCreator gameObjectCreator,
+            GameObject workerGameObject = null)
```

Comment:
Any reason this has a default value of `null` ?

Reply:
Not really while the helper exists. Can remove.